patches/chrome-browser-ui-webui-flags_ui.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/webui/flags_ui.cc b/chrome/browser/ui/webui/flags_ui.cc
+index ad3e0322e50d757e011e68a44286793ceef4c4aa..9c3f1ec44e2fbbc53f3ca997cb8ef929dfabd886 100644
+--- a/chrome/browser/ui/webui/flags_ui.cc
++++ b/chrome/browser/ui/webui/flags_ui.cc
+@@ -218,6 +218,7 @@ void FlagsDOMHandler::HandleEnableExperimentalFeatureMessage(
+       !args->GetString(1, &enable_str))
+     return;
+ 
++  SyncFeatureFlagWithBravePrefs(entry_internal_name, Profile::FromWebUI(web_ui()));
```

Comment:
I'm confused, I thought we were doing a lazy check to sync with the pref?

Reply:
if we need notification at the time this changes then we should add a pref observer for `kEnabledLabsExperiments`

Reply:
I think I could have named it better, the intention of this call is to update the temporary pref `kBraveEnabledMediaRouter` that we use in `brave://settings` when flags are toggled. So toggling the flag correctly toggles the `Media Router` switch in brave://settings.

Reply:
this can be done without a patch using a chromium_src override for `SetFeatureEntryEnabled` which I thought is what we had before and maybe this is why I'm confused here